src/Interpreters/Context.cpp

```diff
@@ -1340,6 +1342,26 @@ BackgroundProcessingPool & Context::getBackgroundPool()
     return *shared->background_pool;
 }
 
+BackgroundProcessingPool & Context::getBackgroundLowPriorityPool()
+{
+    auto lock = getLock();
+    if (!shared->background_low_priority_pool)
+    {
+        BackgroundProcessingPool::PoolSettings pool_settings;
+        const auto & config = getConfigRef();
+        pool_settings.low_cpu_priority = true;
+        pool_settings.thread_sleep_seconds = config.getDouble("background_processing_pool_thread_sleep_seconds", 10);
+        pool_settings.thread_sleep_seconds_random_part = config.getDouble("background_processing_pool_thread_sleep_seconds_random_part", 1.0);
+        pool_settings.thread_sleep_seconds_if_nothing_to_do = config.getDouble("background_processing_pool_thread_sleep_seconds_if_nothing_to_do", 0.1);
+        pool_settings.task_sleep_seconds_when_no_work_min = config.getDouble("background_processing_pool_task_sleep_seconds_when_no_work_min", 10);
+        pool_settings.task_sleep_seconds_when_no_work_max = config.getDouble("background_processing_pool_task_sleep_seconds_when_no_work_max", 600);
+        pool_settings.task_sleep_seconds_when_no_work_multiplier = config.getDouble("background_processing_pool_task_sleep_seconds_when_no_work_multiplier", 1.1);
+        pool_settings.task_sleep_seconds_when_no_work_random_part = config.getDouble("background_processing_pool_task_sleep_seconds_when_no_work_random_part", 1.0);
+        shared->background_low_priority_pool.emplace(settings.background_pool_size, pool_settings);
```

Comment:
Here we use the value that is defined for [regular background pool.](https://github.com/ClickHouse/ClickHouse/blob/master/src/Core/Settings.h#L86)  Is this what's intended?